### ISSUE TITLE
Wrong `no-unused-style` warning with conditional styles

### ISSUE THREAD

**Issue by satya164** [Intellicode/eslint-plugin-react-native#18]
Sample code to reproduce:

```js
import React, { Component, PropTypes } from 'react';
import ReactNative from 'react-native';

const {
	StyleSheet,
	View
} = ReactNative;

const styles = StyleSheet.create({
	style1: {
		color: 'red',
	},
	style2: {
		color: 'blue',
	}
});

export default class MyComponent extends Component {
	static propTypes = {
		isDanger: PropTypes.bool
	};

	render() {
		return <View style={this.props.isDanger ? styles.style1 : styles.style2} />;
	}
}
```

ESLint output:

```sh
/path/to/test.js
  10:2  warning  Unused style detected: styles.style1  react-native/no-unused-styles
  13:2  warning  Unused style detected: styles.style2  react-native/no-unused-styles

✖ 2 problems (0 errors, 2 warnings)
```

ESLint config:

```
    "react-native/no-unused-styles": 1,
    "react-native/split-platform-components": 1,
```

**Comment by Intellicode** [comment]
Thanks for reporting!

**Comment by satya164** [comment]
@Intellicode Awesome!
